### PR TITLE
clarify microbatch per feedback

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr makes updates to the microbatch doc based on [internal slack feedback](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1731976905502389):
- clarify microbatch is diff from other strategies and why
- make note microbatch is new
- add links
- add info that event_time can be used for upstream parents and modify 'example' structure
- add a separate 'how microbatch works' header to break wall of text
- add bullets for easier reading

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-mwong-update-microbatch-dbt-labs.vercel.app/docs/build/incremental-microbatch
- https://docs-getdbt-com-git-mwong-update-microbatch-dbt-labs.vercel.app/docs/build/incremental-models

### CODE REVIEW

File: website/docs/docs/build/incremental-microbatch.md [hunk 1]
```diff
@@ -22,17 +22,32 @@ Refer to [Supported incremental strategies by adapter](/docs/build/incremental-s
 
 Incremental models in dbt are a [materialization](/docs/build/materializations) designed to efficiently update your data warehouse tables by only transforming and loading _new or changed data_ since the last run. Instead of reprocessing an entire dataset every time, incremental models process a smaller number of rows, and then append, update, or replace those rows in the existing table. This can significantly reduce the time and resources required for your data transformations.
 
-Microbatch incremental models make it possible to process transformations on very large time-series datasets with efficiency and resiliency. When dbt runs a microbatch model — whether for the first time, during incremental runs, or in specified backfills — it will split the processing into multiple queries (or "batches"), based on the [`event_time`](/reference/resource-configs/event-time) and `batch_size` you configure.
+Microbatch is a new incremental strategy designed for large time-series datasets:
```
Comment (mirnawong1) [inline website/docs/docs/build/incremental-microbatch.md]:
```suggestion
Microbatch is an incremental strategy designed for large time-series datasets:
```

File: website/docs/docs/build/incremental-microbatch.md [hunk 2]
```diff
@@ -22,17 +22,32 @@ Refer to [Supported incremental strategies by adapter](/docs/build/incremental-s
...
 
-Microbatch incremental models make it possible to process transformations on very large time-series datasets with efficiency and resiliency. When dbt runs a microbatch model — whether for the first time, during incremental runs, or in specified backfills — it will split the processing into multiple queries (or "batches"), based on the [`event_time`](/reference/resource-configs/event-time) and `batch_size` you configure.
+Microbatch is an incremental strategy designed for large time-series datasets:
+- It complements, rather than replaces, existing incremental strategies by focusing on efficiency and simplicity in batch processing.
+- Unlike traditional incremental strategies, microbatch doesn't require managing uniqueness constraints or implementing complex conditional logic for [backfilling](#backfills).
```
Comment (graciegoheen) [inline website/docs/docs/build/incremental-microbatch.md]:
```suggestion
- Unlike traditional incremental strategies, microbatch doesn't require implementing complex conditional logic for [backfilling](#backfills). 
```

We don't do anything special to ensure there are no duplicates - so testing for uniqueness is still a good idea when relevant!

File: website/docs/docs/build/incremental-microbatch.md [hunk 3]
```diff
@@ -22,17 +22,32 @@ Refer to [Supported incremental strategies by adapter](/docs/build/incremental-s
...
 
-Microbatch incremental models make it possible to process transformations on very large time-series datasets with efficiency and resiliency. When dbt runs a microbatch model — whether for the first time, during incremental runs, or in specified backfills — it will split the processing into multiple queries (or "batches"), based on the [`event_time`](/reference/resource-configs/event-time) and `batch_size` you configure.
+Microbatch is an incremental strategy designed for large time-series datasets:
+- It complements, rather than replaces, existing incremental strategies by focusing on efficiency and simplicity in batch processing.
+- Unlike traditional incremental strategies, microbatch doesn't require managing uniqueness constraints or implementing complex conditional logic for [backfilling](#backfills). 
+- It relies solely on a time column ([`event_time`](/reference/resource-configs/event-time)) to handle data partitioning and filtering.
```
Comment (graciegoheen) [inline website/docs/docs/build/incremental-microbatch.md]: `event_time` is technically unique from `partition_by`

also, you set `event_time` column for your microbatch model and its direct parents (might be worth calling out?)

File: website/docs/docs/build/incremental-microbatch.md [hunk 4]
```diff
@@ -22,17 +22,32 @@ Refer to [Supported incremental strategies by adapter](/docs/build/incremental-s
...
+### How microbatch works
+
+When dbt runs a microbatch model — whether for the first time, during incremental runs, or in specified backfills — it will split the processing into multiple queries (or "batches"), based on the `event_time` and `batch_size` you configure.
+
+Each "batch" corresponds to a single bounded time period (by default, a single day of data). Where other incremental strategies operate only on "old" and "new" data, microbatch models treat every batch as an atomic unit that can be built or replaced on its own. Each batch is independent and <Term id="idempotent" />. This is a powerful abstraction that makes it possible for dbt to run batches [separately](#backfills) — in the future, concurrently — and to [retry](#retry) them independently.
```
Comment (graciegoheen) [inline website/docs/docs/build/incremental-microbatch.md]: We just added support for concurrency - see https://github.com/dbt-labs/docs.getdbt.com/issues/6550

File: website/docs/docs/build/incremental-microbatch.md [hunk 5]
```diff
@@ -22,17 +22,32 @@ Refer to [Supported incremental strategies by adapter](/docs/build/incremental-s
...
+- `customers` is a relatively small dimensional table. Customer attributes update often, and not in a time-based manner — that is, older customers are just as likely to change column values as newer customers. The customers model doesn't configure an `event_time` column.
+
+As a result:
 
-The `page_view_start` column in `page_views` is configured as that model's `event_time`. The `customers` model does not configure an `event_time`. Therefore, each batch of `sessions` will filter `page_views` to the equivalent time-bounded batch, and it will not filter `customers` (a full scan for every batch).
+- Each batch of `sessions` will filter `page_views` to the equivalent time-bounded batch.
+- The `customers` table isn't filtered, resulting in a full scan for every batch. 
+
+:::tip
+In addition to configuring `event_time` for the target table, you can also specify it for any upstream models that you want to filter, even if they have different time columns.
```
Comment (graciegoheen) [inline website/docs/docs/build/incremental-microbatch.md]:
```suggestion
In addition to configuring `event_time` for the target table, you should also specify it for any upstream models that you want to filter, even if they have different time columns.
```

File: website/docs/docs/build/incremental-microbatch.md (hunk 4 above)
Comment (mirnawong1) [reply website/docs/docs/build/incremental-microbatch.md]: great stuff, thanks Grace. i've tweaked this and will link out to the 'concurrently' line once we add a section about it and resolve 6550 🙏  that way, i don't block this pr.

https://github.com/dbt-labs/docs.getdbt.com/issues/6550#issuecomment-2503463842

File: website/docs/docs/build/incremental-microbatch.md [hunk 6]
```diff
@@ -22,17 +22,34 @@ Refer to [Supported incremental strategies by adapter](/docs/build/incremental-s
...
 
-Microbatch incremental models make it possible to process transformations on very large time-series datasets with efficiency and resiliency. When dbt runs a microbatch model — whether for the first time, during incremental runs, or in specified backfills — it will split the processing into multiple queries (or "batches"), based on the [`event_time`](/reference/resource-configs/event-time) and `batch_size` you configure.
+Microbatch is an incremental strategy designed for large time-series datasets:
+- It relies solely on a time column ([`event_time`](/reference/resource-configs/event-time)) to define time-based ranges for filtering. Set the `event_time` column for your microbatch model and its direct parents (upstream models). Note, this is different to `partition_by`, which groups rows into partitions.
+- It complements, rather than replaces, existing incremental strategies by focusing on efficiency and simplicity in batch processing.
```
Comment (nataliefiann) [inline website/docs/docs/build/incremental-microbatch.md:27]: Hiya @mirnawong1 

Just to ask, should "complements" be "complements" or "compliments" I wasn't sure if complements is a US spelling.

Kind Regards
Natalie

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Thanks for creating this PR. I've approved this for you with a non-blocking QQ.

Kind Regards
Natalie

File: website/docs/docs/build/incremental-microbatch.md (hunk 6 above)
Comment (mirnawong1) [reply website/docs/docs/build/incremental-microbatch.md:27]: hey @nataliefiann , good question! this is right IMHO as complements means something that enhancements it. compliment generally means praise so in this context, microbatch enhances existing strategies.

**Pull request merged**